unit-tests/func/func-common.h

```diff
@@ -150,3 +192,26 @@ inline void do_while_streaming( rs2::sensor depth_sens,
     depth_sens.stop();
     depth_sens.close();
 }
+
+
+rs2::stream_profile get_profile_by_stream_parameters(
+    rs2::sensor s, rs2_stream stream, rs2_format format, int width = 640, int height = 480, int fps = 30, int stream_index = 0 )
```

Comment:
I don't think the width/height/fps/stream_index should have defaults unless they mean 'any'

Reply:
Most of the time you don't care about them so it is nice to not set them and automatically it works :)

But I removed it anyway.